src/main/java/clueless/Game.java

```diff
@@ -383,15 +384,38 @@ public void loseGame(Player loser) {
 	 * @return
 	 */
 	public Player whoHasClue(List<Card> suggestion) {
-		Player pl;
+		Player pl = null;
```

Comment:
rather that initializing and assigning to null and returning `pl` at the end, just initialize it here and if you make it to the end of method return null